clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5583,6 +5583,9 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
       CmdArgs.push_back("-fsycl-allow-func-ptr");
     }
 
+    Args.AddLastArg(CmdArgs, options::OPT_fsycl_rtc_mode,
+                    options::OPT_fno_sycl_rtc_mode);
```

Comment:
Can you add a small driver test to verify the passing of this to the expected `-cc1` phase?

Reply:
Sure, no problem. I had looked around before posting the PR to see if other options handled similarly are tested in a certain manner, but didn't find a good example (probably me being blind). Do you happen to have a example of a good test for that?

Reply:
I think test/Driver/sycl-xarch.cpp has a good example.  The first test in the file has a check for the device compilation confirming the needed option is passed in that command.

Reply:
Thanks for the hint, I've added a test now.